src/frontends/paddle/src/op/linspace.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/frontend/paddle/visibility.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs linspace(const NodeContext& node) {
+    auto start = node.get_input("Start");
+    auto stop = node.get_input("Stop");
+    auto num = node.get_input("Num");
+    std::unordered_map<int, element::Type> umap{{2, element::i32}, {3, element::i64}, {5, element::f32}};
+    const auto dtype = node.get_attribute<int>("dtype", 5);
```

Comment:
can we use ```auto dtype = node.get_attribute<ov::element::Type>("dtype");``` to get the the type ?

Reply:
Done

Reply:
Pls add a screenshot of test results, thanks